docs/management/connectors/action-types/cases-webhook.asciidoc

```diff
@@ -0,0 +1,179 @@
+[role="xpack"]
+[[cases-webhook-action-type]]
+=== Webhook - Case Management connector and action
+++++
+<titleabbrev>Webhook - Case Management</titleabbrev>
+++++
+
+The Webhook - Case Management connector uses https://github.com/axios/axios[axios] to send POST, PUT, and GET requests to a Case Management RESTful API web service. The resulting connector can be used with Kibana Cases to create and update a case in the third party web service.
+
+[float]
+[[webhook-connector-configuration]]
+==== Connector configuration
+
+Webhook - Case Management connectors have the following configuration properties.
+
+Name::      The name of the connector. The name is used to identify a  connector in the management UI connector listing, or in the connector list when configuring an action.
+Require authentication:: If true, a username and password for login type authentication must be provided.
+Username::      Username for HTTP basic authentication.
+Password::  Password for HTTP basic authentication.
+Headers::   A set of key-value pairs sent as headers with the request
```

Comment:
I think it is valuable to mention, that this headers will be added to each of the URLs for create, update and comment.